pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -82,6 +85,10 @@ mkDerivation rec {
   # TODO: Package mapbox-variant
 
   postFixup = ''
+    # Nuke refs to `tg_owt` which is introduced by `__FILE__` in headers.
+    sed -E "s|($NIX_STORE/)[a-z0-9]{32}(-${tg_owt.name})|\1eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee\2|g" \
+      --in-place $out/bin/telegram-desktop
+
```

Comment:
There is `removeReferencesTo` function for that

Reply:
Awesome, thanks for bringing that up! I was hoping something like that would exist but only found `nuke-refs` from a quick search :o We'll change that soon then :)